docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -0,0 +1,187 @@
+
+A conforming implementation of this protocol may provide additional functionality, but must not where explicitly disallowed or would otherwise result in non‐conformance.
+
+### Overview
+
+An _update_ is defined as a [_manifest_](#manifest-response) together with the assets referenced inside the manifest.
+Expo Updates is a protocol for delivering updates to apps running on multiple platforms.
+
+An app running a conformant Expo Updates client-library will fetch the most recent update from a conformant update server. If the client-library cannot fetch an update or if it already has the most recent update, the client-library will load the update saved in the client-library's cache.
+
+The following describes how a client-library will interact with a conforming updates server:
+1. The client-library will make a [request](#request) for the most recent manifest. 
+2. If it is a new manifest, the client-library will proceed to make requests to download and store the assets specified in the manifest.
+3. Finally, the client-library will update its local state according to any metadata provided by the response [headers](#headers).
+
+We anticipate the primary user of this spec will be organizations who need to manage their own update server to satisfy internal requirements.
+
+## Manifest Request
+A conformant client-library MUST make a GET request with the following headers:
+```
+accept: application/json
+expo-platform: string
+expo-runtime-version: string
+```
+along with any headers stipulated by a previous responses' [server defined headers](#headers):
+
+* `expo-platform` MUST be platform type the client is running on: 
+    * iOS MUST be `expo-platform: ios`
+    * Android MUST be `expo-platform: android`
+* `expo-runtime-version` MUST be the runtime version the client is running on.
+
+A conformant client-library SHOULD also accept `application/expo+json` so that servers can easily tell when to send back an Expo manifest:
+```
+accept: application/expo+json, application/json
+```
+
+## Manifest Response
+
+A conformant server MUST return a body containing a [manifest](#body) along with the [headers](#headers) specified below.
+
+The choice of manifest and headers are dependent on the values of the request headers. A conformant server MUST:
+
+* Return a manifest that describes an update capable of running on the platform and runtime version specified in the `expo-platform` and `expo-runtime-version` request headers. 
+* Return a manifest that describes the most recent, sorted by creation time, update satisfying any constraints imposed by the headers.
+
+### Headers
+
+```
+expo-protocol-version: 0
+expo-sfv-version: 0
+expo-manifest-filters: expo-sfv
+expo-server-defined-headers: expo-sfv
+cache-control: *
+content-type: application/json; charset=utf-8
```

Comment:
one thing that's not clear is which values are literals and which are types

```suggestion
expo-manifest-filters: <expo-sfv>
expo-server-defined-headers: <expo-sfv>
cache-control: <*>
content-type: application/expo+json; charset=utf-8
```

also, if the client accepts `application/expo+json`, the server may want to be precise and respond with `application/expo+json`. the server should follow the http spec and send back a value compliant with the request's accept header or a 406 if applicable